### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by Redredemption9** [tropicbliss/buckshot#18]
I literally have 0 clue how I can use this I've been using MCSniperPY which was simple as putting a command in then the offset and the name that wanted to be sniped along with the account, if you can dumb it down for me and tell me the command I would put in to launch buckshot it'd be much appreciated. and if you don't know what I mean I mean as an example I'll use the MCSniperPY command : py -m mcsniperpy snipe
Something along the lines of this I need help 
Thanks.

**Comment by tropicbliss** [comment]
I agree that this is slightly more difficult to set up. Honestly if I were you I would use MCsniperGO instead which should be better than MCsniperPY. You could ask questions in their discord and since the instructions are almost copy and pasted from MCsniperGO instructions you should be able to use this too.

Yeah I realise this is not as user friendly as you would like but it is more meant for my personal use. You would not be missing much using MCsniperGO over this sniper (which tbf imo has a better success rate than MCsniperGO)

**Comment by Redredemption9** [comment]
I just mainly wanted to use something like this sniper for gc sniping as I have no clue how to with MCSniperGO

**Comment by tropicbliss** [comment]
Just ask on their discord. I guarantee you it's easier than this sniper

**Comment by Redredemption9** [comment]
I tried finding it and I couldn’t for MCSniperGO or MCSniperPY

**Comment by Redredemption9** [comment]
Nevermind I’m slow I found it.
